test_scripts/Smoke/Resumption/ATF_Resumption_3rd_ignition_cycle.lua

```diff
@@ -0,0 +1,161 @@
+--  Requirement summary:
+--  [Data Resumption]: Data resumption on IGNITION OFF
+--
+--  Description:
+--  Check that SDL does not perform App resumption in case when media app tries to resume in 3rd ignition cycle.
+--
+--  1. Used precondition
+--  Media App is registered and active on HMI
+--
+--  2. Performed steps
+--  Send IGNITION_OFF from HMI.
+--  Start SDL. (2nd ignition cycle)
+--  Send IGNITION_OFF from HMI.
+--  Start SDL. (3rd ignition cycle)
+--  Connect transport.
+--
+--  Expected behavior:
+--  1. In 3rd ignition cycle App is registered and get default HMI level, app data is resumed.
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app = nil -- will be initialized after application registration
+
+-- [[ Local Functions ]]
+local function IGNITION_OFF()
+  Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "SUSPEND" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLPersistenceComplete")
+
+  Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "IGNITION_OFF" })
+  EXPECT_NOTIFICATION("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = false }) 
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose"):Do(function ()
+    StopSDL()
+  end)
+end
+
+local function Restart_SDL_And_Add_Mobile_Connection()
+  Test:runSDL()
+  commonFunctions:waitForSDLStart(Test):Do(function()
+    Test:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      Test:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        Test:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
```

Comment:
@LitvinenkoIra There is function Restart_SDL_And_Add_Mobile_Connection(), why it isn't used here?